app/src/main/res/values/strings.xml

```diff
@@ -7,6 +7,7 @@
     <string name="all_today">today</string>
     <string name="all_yesterday">yesterday</string>
     <string name="all_date_and_time">%1$s at %2$s</string>
+    <string name="all_date_range" translatable="false">%1$s\u2013%2$s</string>
```

Comment:
Good point, but soft hyphen (or other chars) do not seem to work very well because textview will hyphenate on the numbers and dots, so I might look at this later, and keep this as is for now?

Reply:
Yep this is good enough for me too, preferably just write this down somewhere.